release/k3s/k3s.go

```diff
@@ -80,6 +80,7 @@ type Release struct {
 	Handler       string `json:"handler"`
 	Email         string `json:"email"`
 	Token         string `json:"token"`
+	SSHKeyPath    string `json:"ssh_key_path"`
```

Comment:
If this is now a mandatory field, this needs to be able to fail if it's not present in the config.

Reply:
There is fallback condition currently builtin to the getAuth function that specifies the hard-coded path. There is currently no config checking within the cli package. I could replace the existing fallback with an error.

Reply:
Link to the `getAuth` function please. Either way, we need to communicate expectations around this implied behavior. So a doc, a comment, or something should be made somewhere.

Reply:
Let's do that and update any references that this code has to handle the error and communicate it back to the caller.